src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -21,27 +23,39 @@ interface DotEditPageNavItem {
     templateUrl: './dot-edit-page-nav.component.html',
     styleUrls: ['./dot-edit-page-nav.component.scss']
 })
-export class DotEditPageNavComponent implements OnInit {
+export class DotEditPageNavComponent implements OnChanges {
     @Input() pageState: DotRenderedPageState;
     model: Observable<DotEditPageNavItem[]>;
+    isEnterpriseLicense: boolean;
 
     constructor(private dotLicenseService: DotLicenseService, public dotMessageService: DotMessageService, public route: ActivatedRoute) {}
 
-    ngOnInit(): void {
-        this.model = this.dotMessageService
+    ngOnChanges(changes: SimpleChanges): void {
+        if (changes.pageState.currentValue !== changes.pageState.previousValue) {
```

Comment:
You sure you need to do this change? because as far as I understand if the ```ngOnChanges``` trigger is because a property change, right?